runtime/parachains/src/scheduler/tests.rs

```diff
@@ -142,20 +147,22 @@ fn add_parathread_claim_works() {
 
 		assert!(Paras::is_parathread(thread_id));
 
-		let assignment = Assignment::ParathreadA(ParathreadEntry {
-			claim: ParathreadClaim(thread_id, Some(collator.clone())),
-			retries: 0,
-		});
-		let ca = assignment.to_core_assignment(core_index, group_index);
-		Scheduler::add_to_claimqueue(ca.clone());
+		let pe = ParasEntry::new(Assignment::new(
+			thread_id,
+			CollatorRestrictions::new(
+				[OpaquePeerId::new(collator.to_raw_vec())].into_iter().collect(),
+				CollatorRestrictionKind::Preferred,
+			),
+		));
+		Scheduler::add_to_claimqueue(core_index, pe.clone());
 
 		let cq = Scheduler::claimqueue();
 		assert_eq!(Scheduler::claimqueue_len(), 1);
-		assert_eq!(*(cq.get(&core_index).unwrap().front().unwrap()), Some(ca));
+		assert_eq!(*(cq.get(&core_index).unwrap().front().unwrap()), Some(pe));
 	})
 }
 
-/// MOVE TO polkadot provider
+/// TODO: MOVE TO polkadot provider
```

Comment:
Ticket or do it, otherwise it will never be done.

Reply:
Edited https://github.com/orgs/paritytech/projects/67/views/1?pane=issue&itemId=26427074 to include the move.